packages/yew/src/functional/hooks/use_state.rs

```diff
@@ -122,6 +122,14 @@ impl<T> UseStateHandle<T> {
             inner: self.inner.dispatcher(),
         }
     }
+
+    /// Returns the current value.
```

Comment:
I think "current" is a bit misleading here. The captured `value` will be the value that state had during the view call and not update. Test the following:

```rust
let value = use_state(|| 42);
if value.get() == 42 {
    value.set(0); // This will trigger a re-render, but the value in this `UseStateHandle`
    assert_eq!(value.get(), 42); // will continue to return 42
}
```